lesson28/src/js/changepassword.js

```diff
@@ -40,4 +42,19 @@ const togglePasswordDisplay = target => {
     target.classList.toggle("is-open");
 }
 
-submitButton.addEventListener("click", () => console.log("submitされました。この後はまだ未実装です。"))
+const changeRegisteredDataOfPassword = () => {
+    const newPassword = passwordOfInput.value;
+    const userData = JSON.parse(localStorage.getItem("registeredData"));
+    userData.password = newPassword;
+    localStorage.setItem("registeredData", JSON.stringify(userData));
+}
+
+submitButton.addEventListener("click", () => {
+    changeRegisteredDataOfPassword();
+
+    const newToken = chance.apple_token();
```

Comment:
```suggestion
    const token = chance.apple_token();
```
newの比較であるoldがないので、あまり意識しないでいい気がします

Reply:
ご指摘ありがとうございます！
こちらもその通りだと思いました。
このJSファイルの中で、他にトークンについて言及している部分がありませんでした。
tokenに変更いたしました。
[ec88478](https://github.com/haru-programming/JavaScript-lessons/pull/36/commits/ec88478f8eb3de43c69323e67e2693b80884e85b)